Makefile.include

```diff
@@ -95,6 +95,20 @@ __DIRECTORY_VARIABLES += \
 
 BOARDSDIR               := $(abspath $(BOARDSDIR))
 
+# Include Board and CPU configuration, if BOARD is not found in BOARDSDIR
+# e.g. when set by the environment fallback to searching in RIOTBOARD
+ifeq (,$(wildcard $(BOARDSDIR)/$(BOARD)))
+  ifneq ($(RIOTBOARD),$(BOARDSDIR))
+    # The specified board $(BOARD) was not found in $(BOARDSDIR) fallback to RIOTBOARD)
```

Comment:
Note, I didn't use a warning or info here since a lot of the code using `info-boards-supported` expects a clean output, only the boardslist (e.g. `compile_and_test_for_board`).